core/lib/multivm/src/tracers/call_tracer/mod.rs

```diff
@@ -38,4 +52,16 @@ impl CallTracer {
         let cell = self.result.as_ref();
         cell.set(result).unwrap();
     }
+
+    fn push_call_and_update_stats(&mut self, call: FarcallAndNearCallCount) {
+        self.stack.push(call);
+        self.max_stack_depth = self.max_stack_depth.max(self.stack.len());
```

Comment:
we have to include near calls as well

Reply:
but near calls are updated in the other method, when the actual increasing if value occurs

Reply:
Ah, okay, I understand your point

Reply:
I mean, that near call is also part of the actual depth.
We just don't add recursion here because it can cause stack overflow later

Reply:
Got it, then my previous thought was a bit wrong :)